Self-reveiw

ErisenoLuv opened the pull request:
__
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [types-of-contributions.md](/main/contributing/types-of-contributions.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes [issue link]

<!--
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. If you made changes to the `content` directory, a table will populate in a comment below with the staging and live article links -->

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->
__

ErisenoLuv commented:
Reach me @ hislaragirl12@gmail.com for any feedback or changes.

Princeprince559 commented:
I'm pretty sure I don't know how to do that. I'm sorry

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/ghei36>
________________________________
From: ErisenoLuv ***@***.***>
Sent: Tuesday, February 15, 2022 5:22:46 PM
To: github/docs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [github/docs] Self-reveiw (PR #15519)


__
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our CONTRIBUTING.md</main/CONTRIBUTING.md> for information how to contribute.

For changes to content in site policy<https://github.com/github/docs/tree/main/content/github/site-policy>, see the CONTRIBUTING guide in the site-policy repo<https://github.com/github/site-policy/blob/main/CONTRIBUTING.md>.

We cannot accept changes to our translated content right now. See the types-of-contributions.md</main/contributing/types-of-contributions.md#earth_asia-translations> for more information.

Thanks again!
-->

Why:

Closes [issue link]

What's being changed:
Check off the following:

  *    I have reviewed my changes in staging (look for "Automatically generated comment" and click Modified to view your latest changes).
  *    For content changes, I have completed the self-review checklist<https://github.com/github/docs/blob/main/contributing/self-review.md#self-review>.

Writer impact (This section is for GitHub staff members only):

  *    This pull request impacts the contribution experience
     *    I have added the 'writer impact' label
     *    I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

__

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/github/docs/pull/15519

Commit Summary

  *   5d91fd3<https://github.com/github/docs/pull/15519/commits/5d91fd35a15982933efd65a8f038b4b2410d96e3> Self-reveiw

File Changes

(1 file<https://github.com/github/docs/pull/15519/files>)

  *   M contributing/self-review.md<https://github.com/github/docs/pull/15519/files#diff-4eb3fe3bb17e371765bc1f0a8787415a72b2ab515aa4ff97fc9339e1888c387f> (14)

Patch Links:

  *   https://github.com/github/docs/pull/15519.patch
  *   https://github.com/github/docs/pull/15519.diff

—
Reply to this email directly, view it on GitHub<https://github.com/github/docs/pull/15519>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AV7ADWZEXWVLBJ3UBVMDPF3U3L36NANCNFSM5OQFY5IQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

ErisenoLuv commented:
So what's going on? What changes can't you access or not allowed to do? I
may be a help on helping with the situation

On Tue, Feb 15, 2022, 6:24 PM Princeprince559 ***@***.***>
wrote:

> I'm pretty sure I don't know how to do that. I'm sorry
>
> Sent from my T-Mobile 4G LTE Device
> Get Outlook for Android<https://aka.ms/ghei36>
> ________________________________
> From: ErisenoLuv ***@***.***>
> Sent: Tuesday, February 15, 2022 5:22:46 PM
> To: github/docs ***@***.***>
> Cc: Subscribed ***@***.***>
> Subject: [github/docs] Self-reveiw (PR #15519)
>
>
> __
> Thank you for contributing to this project! You must fill out the
> information below before we can review this pull request. By explaining why
> you're making a change (or linking to a pull request) and what changes
> you've made, we can triage your pull request to the best possible team for
> review.
>
> See our CONTRIBUTING.md</main/CONTRIBUTING.md> for information how to
> contribute.
>
> For changes to content in site policy<
> https://github.com/github/docs/tree/main/content/github/site-policy>, see
> the CONTRIBUTING guide in the site-policy repo<
> https://github.com/github/site-policy/blob/main/CONTRIBUTING.md>.
>
> We cannot accept changes to our translated content right now. See the
> types-of-contributions.md</main/contributing/types-of-contributions.md#earth_asia-translations>
> for more information.
>
> Thanks again!
> -->
>
> Why:
>
> Closes [issue link]
>
> What's being changed:
> Check off the following:
>
> * I have reviewed my changes in staging (look for "Automatically generated
> comment" and click Modified to view your latest changes).
> * For content changes, I have completed the self-review checklist<
> https://github.com/github/docs/blob/main/contributing/self-review.md#self-review
> >.
>
> Writer impact (This section is for GitHub staff members only):
>
> * This pull request impacts the contribution experience
> * I have added the 'writer impact' label
> * I have added a description and/or a video demo of the changes below
> (e.g. a "before and after video")
>
> __
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/github/docs/pull/15519
>
> Commit Summary
>
> * 5d91fd3<
> https://github.com/github/docs/pull/15519/commits/5d91fd35a15982933efd65a8f038b4b2410d96e3>
> Self-reveiw
>
> File Changes
>
> (1 file<https://github.com/github/docs/pull/15519/files>)
>
> * M contributing/self-review.md<
> https://github.com/github/docs/pull/15519/files#diff-4eb3fe3bb17e371765bc1f0a8787415a72b2ab515aa4ff97fc9339e1888c387f>
> (14)
>
> Patch Links:
>
> * https://github.com/github/docs/pull/15519.patch
> * https://github.com/github/docs/pull/15519.diff
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/github/docs/pull/15519>, or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AV7ADWZEXWVLBJ3UBVMDPF3U3L36NANCNFSM5OQFY5IQ
> >.
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/15519#issuecomment-1040995360>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXZFQ2KXEJG2MFWEBN6F3ITU3L4FTANCNFSM5OQFY5IQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Princeprince559 commented:
I'm sure I have total control... just not sure of any of it... I'm new to this

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/ghei36>
________________________________
From: ErisenoLuv ***@***.***>
Sent: Tuesday, February 15, 2022 5:27:39 PM
To: github/docs ***@***.***>
Cc: Princeprince559 ***@***.***>; Comment ***@***.***>
Subject: Re: [github/docs] Self-reveiw (PR #15519)


So what's going on? What changes can't you access or not allowed to do? I
may be a help on helping with the situation

On Tue, Feb 15, 2022, 6:24 PM Princeprince559 ***@***.***>
wrote:

> I'm pretty sure I don't know how to do that. I'm sorry
>
> Sent from my T-Mobile 4G LTE Device
> Get Outlook for Android<https://aka.ms/ghei36>
> ________________________________
> From: ErisenoLuv ***@***.***>
> Sent: Tuesday, February 15, 2022 5:22:46 PM
> To: github/docs ***@***.***>
> Cc: Subscribed ***@***.***>
> Subject: [github/docs] Self-reveiw (PR #15519)
>
>
> __
> Thank you for contributing to this project! You must fill out the
> information below before we can review this pull request. By explaining why
> you're making a change (or linking to a pull request) and what changes
> you've made, we can triage your pull request to the best possible team for
> review.
>
> See our CONTRIBUTING.md</main/CONTRIBUTING.md> for information how to
> contribute.
>
> For changes to content in site policy<
> https://github.com/github/docs/tree/main/content/github/site-policy>, see
> the CONTRIBUTING guide in the site-policy repo<
> https://github.com/github/site-policy/blob/main/CONTRIBUTING.md>.
>
> We cannot accept changes to our translated content right now. See the
> types-of-contributions.md</main/contributing/types-of-contributions.md#earth_asia-translations>
> for more information.
>
> Thanks again!
> -->
>
> Why:
>
> Closes [issue link]
>
> What's being changed:
> Check off the following:
>
> * I have reviewed my changes in staging (look for "Automatically generated
> comment" and click Modified to view your latest changes).
> * For content changes, I have completed the self-review checklist<
> https://github.com/github/docs/blob/main/contributing/self-review.md#self-review
> >.
>
> Writer impact (This section is for GitHub staff members only):
>
> * This pull request impacts the contribution experience
> * I have added the 'writer impact' label
> * I have added a description and/or a video demo of the changes below
> (e.g. a "before and after video")
>
> __
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/github/docs/pull/15519
>
> Commit Summary
>
> * 5d91fd3<
> https://github.com/github/docs/pull/15519/commits/5d91fd35a15982933efd65a8f038b4b2410d96e3>
> Self-reveiw
>
> File Changes
>
> (1 file<https://github.com/github/docs/pull/15519/files>)
>
> * M contributing/self-review.md<
> https://github.com/github/docs/pull/15519/files#diff-4eb3fe3bb17e371765bc1f0a8787415a72b2ab515aa4ff97fc9339e1888c387f>
> (14)
>
> Patch Links:
>
> * https://github.com/github/docs/pull/15519.patch
> * https://github.com/github/docs/pull/15519.diff
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/github/docs/pull/15519>, or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AV7ADWZEXWVLBJ3UBVMDPF3U3L36NANCNFSM5OQFY5IQ
> >.
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/15519#issuecomment-1040995360>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXZFQ2KXEJG2MFWEBN6F3ITU3L4FTANCNFSM5OQFY5IQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

—
Reply to this email directly, view it on GitHub<https://github.com/github/docs/pull/15519#issuecomment-1040997064>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AV7ADW7DSQX4MJSPT544GHTU3L4QXANCNFSM5OQFY5IQ>.
You are receiving this because you commented.Message ID: ***@***.***>

ErisenoLuv commented:
Okay, I totally understand and yes I agree with you. Well have you decided
on a project to start building?

On Tue, Feb 15, 2022, 6:31 PM Princeprince559 ***@***.***>
wrote:

> I'm sure I have total control... just not sure of any of it... I'm new to
> this
>
> Sent from my T-Mobile 4G LTE Device
> Get Outlook for Android<https://aka.ms/ghei36>
> ________________________________
> From: ErisenoLuv ***@***.***>
> Sent: Tuesday, February 15, 2022 5:27:39 PM
> To: github/docs ***@***.***>
> Cc: Princeprince559 ***@***.***>; Comment ***@***.***>
> Subject: Re: [github/docs] Self-reveiw (PR #15519)
>
>
> So what's going on? What changes can't you access or not allowed to do? I
> may be a help on helping with the situation
>
> On Tue, Feb 15, 2022, 6:24 PM Princeprince559 ***@***.***>
> wrote:
>
> > I'm pretty sure I don't know how to do that. I'm sorry
> >
> > Sent from my T-Mobile 4G LTE Device
> > Get Outlook for Android<https://aka.ms/ghei36>
> > ________________________________
> > From: ErisenoLuv ***@***.***>
> > Sent: Tuesday, February 15, 2022 5:22:46 PM
> > To: github/docs ***@***.***>
> > Cc: Subscribed ***@***.***>
> > Subject: [github/docs] Self-reveiw (PR #15519)
> >
> >
> > __
> > Thank you for contributing to this project! You must fill out the
> > information below before we can review this pull request. By explaining
> why
> > you're making a change (or linking to a pull request) and what changes
> > you've made, we can triage your pull request to the best possible team
> for
> > review.
> >
> > See our CONTRIBUTING.md</main/CONTRIBUTING.md> for information how to
> > contribute.
> >
> > For changes to content in site policy<
> > https://github.com/github/docs/tree/main/content/github/site-policy>,
> see
> > the CONTRIBUTING guide in the site-policy repo<
> > https://github.com/github/site-policy/blob/main/CONTRIBUTING.md>.
> >
> > We cannot accept changes to our translated content right now. See the
> >
> types-of-contributions.md</main/contributing/types-of-contributions.md#earth_asia-translations>
> > for more information.
> >
> > Thanks again!
> > -->
> >
> > Why:
> >
> > Closes [issue link]
> >
> > What's being changed:
> > Check off the following:
> >
> > * I have reviewed my changes in staging (look for "Automatically
> generated
> > comment" and click Modified to view your latest changes).
> > * For content changes, I have completed the self-review checklist<
> >
> https://github.com/github/docs/blob/main/contributing/self-review.md#self-review
> > >.
> >
> > Writer impact (This section is for GitHub staff members only):
> >
> > * This pull request impacts the contribution experience
> > * I have added the 'writer impact' label
> > * I have added a description and/or a video demo of the changes below
> > (e.g. a "before and after video")
> >
> > __
> >
> > ________________________________
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/github/docs/pull/15519
> >
> > Commit Summary
> >
> > * 5d91fd3<
> >
> https://github.com/github/docs/pull/15519/commits/5d91fd35a15982933efd65a8f038b4b2410d96e3
> >
> > Self-reveiw
> >
> > File Changes
> >
> > (1 file<https://github.com/github/docs/pull/15519/files>)
> >
> > * M contributing/self-review.md<
> >
> https://github.com/github/docs/pull/15519/files#diff-4eb3fe3bb17e371765bc1f0a8787415a72b2ab515aa4ff97fc9339e1888c387f
> >
> > (14)
> >
> > Patch Links:
> >
> > * https://github.com/github/docs/pull/15519.patch
> > * https://github.com/github/docs/pull/15519.diff
> >
> > —
> > Reply to this email directly, view it on GitHub<
> > https://github.com/github/docs/pull/15519>, or unsubscribe<
> >
> https://github.com/notifications/unsubscribe-auth/AV7ADWZEXWVLBJ3UBVMDPF3U3L36NANCNFSM5OQFY5IQ
> > >.
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/github/docs/pull/15519#issuecomment-1040995360>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXZFQ2KXEJG2MFWEBN6F3ITU3L4FTANCNFSM5OQFY5IQ
> >
> > .
> > Triage notifications on the go with GitHub Mobile for iOS
> > <
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> >
> > or Android
> > <
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> >
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/github/docs/pull/15519#issuecomment-1040997064>, or
> unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AV7ADW7DSQX4MJSPT544GHTU3L4QXANCNFSM5OQFY5IQ
> >.
> You are receiving this because you commented.Message ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/15519#issuecomment-1040999105>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXZFQ2MTNMUFTP5C6YV4ST3U3L45RANCNFSM5OQFY5IQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
